spec/requests/atmosphere/api/user_keys_spec.rb

```diff
@@ -184,12 +181,8 @@
         }
         post api("/user_keys", user), invalid_user_key_req
         expect(response.status).to eq 422
-        expect(json_response['details']).to eq({
-          'public_key' => [
-            'bad type of key (only ssh-rsa is allowed)',
-            'is invalid'
-          ]
-        })
+        expect(json_response['details']['public_key']).
+          to include('bad type of key (only ssh-rsa is allowed)')
```

Comment:
Are you sure that hash equals can be replaced with include? The latter does not verify that other errors are not present? Just wondering if it's a good practise...